common/federation/src/test/java/app/coronawarn/server/common/federation/client/upload/BatchUploadResponseTest.java

```diff
@@ -17,4 +17,18 @@ void checkEmptyBatchUploadResponse() {
     assertThat(batchUploadResponse.getStatus409()).isEmpty();
     assertThat(batchUploadResponse.getStatus500()).isEmpty();
   }
+
+  @Test 
+  void toStringTestBatchUploadResponse(){
+    List<String> status = Arrays.asList("test");
+    BatchUploadResponse batchUploadResponse = new BatchUploadResponse(status,status,status);
+    assertThat(batchUploadResponse.toString()).isEqualTo("BatchUploadResponse{"
+    + "status409="
+    + "test"
+    + ", status500="
+    + "test"
+    + ", status201="
+    + "test"
+    + '}';)
```

Comment:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:testCompile (default-testCompile) on project federation: Compilation failure: Compilation failure:
[ERROR] /root/cwa-server/common/federation/src/test/java/app/coronawarn/server/common/federation/client/upload/BatchUploadResponseTest.java:[32,10] ')' expected
[ERROR] /root/cwa-server/common/federation/src/test/java/app/coronawarn/server/common/federation/client/upload/BatchUploadResponseTest.java:[32,11] illegal start of expression